utils/twarc-archive.py

```diff
@@ -44,9 +44,9 @@
 def main():
     e = os.environ.get
     parser = argparse.ArgumentParser("archive")
-    parser.add_argument("search", action="store",
+    parser.add_argument("search", type=commandline_unicode, action="store",
```

Comment:
I'm sorry, my bad. Made a stupid mistake not including a needed function. 


#Function to decode args to unicode
def commandline_unicode(bytestring):
    unicode_string = bytestring.decode(sys.getfilesystemencoding())
    return unicode_string